packages/account/src/Components/forms/form-select-field.tsx

```diff
@@ -0,0 +1,82 @@
+import { Autocomplete, SelectNative } from '@deriv/components';
+import { useStore } from '@deriv/stores';
+import { Field, FieldProps, FormikErrors } from 'formik';
+import React from 'react';
+import { TGetField, TListItem } from '../additional-kyc-info-modal/form-config';
+
+type TProps = TGetField & {
+    onItemSelection?: (item: TListItem) => void;
+    list_height?: string;
+};
+
+const FormSelectField: React.FC<TProps> = ({
+    label,
+    name,
+    required = false,
+    disabled = false,
+    list_items,
+    onItemSelection,
+    placeholder,
+    list_height,
+}) => {
+    const { ui } = useStore();
+    const { is_mobile } = ui;
+
+    const onSelect =
+        (
+            field: string,
+            setFieldValue: (
+                field: string,
+                value: string,
+                shouldValidate?: boolean
+            ) => Promise<void | FormikErrors<Record<string, string>>>
+        ) =>
```

Comment:
remove the inline typing for a better readability.. the type is lengthier than the function definition 😄